_1327/templates/base.html

```diff
@@ -95,6 +95,18 @@
 							</ul>
 						</li>
 					{% endif %}
+					{# Search form #}
+					<li class ="pull-right">
+						<a href="#" id="popover" data-placement="bottom">
```

Comment:
please make the popover dismissable (http://getbootstrap.com/docs/4.1/components/popovers/#dismiss-on-next-click)